samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -881,4 +922,26 @@ void BLEConnectivityManager::Recovery::StartTimer()
 	k_timer_start(&mRecoveryTimer, K_SECONDS(time), K_NO_WAIT);
 }
 
+BLEConnectivityManager::State BLEConnectivityManager::GetCurrentState()
+{
+	/* Iterate through states from the maximum priority to the minimum one */
+	for (uint8_t s = State::Max; s < State::Min; s++) {
+		if (mStateBitmask & (1 << s)) {
+			return static_cast<State>(s);
+		}
+	}
+	return State::Unknown;
+}
+
+void BLEConnectivityManager::UpdateStateFlag(State newState, bool enabled)
```

Comment:
If function updates the given state, i would not use `newState` name but rather just a `state` name.